sahi/predict.py

```diff
@@ -314,7 +316,11 @@ def predict(
     postprocess_match_metric: str = "IOS",
     postprocess_match_threshold: float = 0.5,
     postprocess_class_agnostic: bool = False,
-    export_visual: bool = False,
+    export_visual: bool = True,
+    export_video: bool = False,
+    view_image: bool = False,
+    video_path: str = None,
```

Comment:
lets remove this var and take the video path with `source` parameter

Reply:
I'll do it in a few days.